auth0/structure_auth0_tenant.go

```diff
@@ -93,6 +93,19 @@ func flattenTenantUniversalLogin(universalLogin *management.TenantUniversalLogin
 	return []interface{}{m}
 }
 
+func flattenTenantSessionCookie(sessionCookie *management.TenantSessionCookie) []interface{} {
+	if sessionCookie == nil {
+		return nil
+	}
+	if sessionCookie.Mode == nil {
+		return nil
+	}
+	m := make(map[string]interface{})
+	m["mode"] = sessionCookie.Mode
+
+	return []interface{}{m}
+}
```

Comment:
These checks are done here https://github.com/auth0/go-auth0/blob/cce36896b06cf5828a5a1da333d877e13bb4e0b1/management/management.gen.go#L6052-L6058 so you can simply use `sessionCookie.GetMode()`